tests/test_tti.py

```diff
@@ -36,10 +36,10 @@ def test_tti(dimensions, space_order):
 
     model = Model(origin, spacing, dimensions,
                   true_vp, nbpml=nbpml,
-                  epsilon=0.0 * true_vp,
-                  delta=0.0 * true_vp,
-                  theta=0.0 * true_vp,
-                  phi=0.0 * true_vp)
+                  epsilon=None,
```

Comment:
It is, but don;t want people to have to dig into Model to see it, this is still TTI and wanna make sure it shows at setup.